### DISCUSSION TITLE
See postgresql-triggers in DBeaver

### DISCUSSION THREAD

**Discussion by ridicolos** [dbeaver/dbeaver#14549] [Q&A]
Hey,

I am currently playing around with triggers on my postgresql-server. After creating a trigger I wanted to see a list of all the triggers I created. Unfortunately I wasn't able to find any menu inside DBeaver to display them.
Is this not supported? Am I just blind?

With psql and a simple \dy I can see a list of all my triggers. But I couldn't find it inside DBeaver.

**Comment by LonwoLonwo** [comment]
Hello @ridicolos 

What Dbeaver version do you use?

Can you check your connection view settings? Triggers will be available in Advanced and Custom mode.

![2021-11-15 15_21_08-DBeaver 21 2 5 - _localhost 3_ Script-22](https://user-images.githubusercontent.com/45152336/141781303-72337740-02a0-4c4e-9ce1-a7ecae950df6.png)

![2021-11-15 17_01_51-DBeaver Ultimate 21 2 5 - _postgres12local_ Script-167](https://user-images.githubusercontent.com/45152336/141794893-182be775-3dac-4e3c-a76e-a55eb5a87a4a.png)

**Reply by ridicolos** [reply]
Hey @LonwoLonwo, now I can see normal triggers. Thanks for that! :)

Do you also know how I can see event-triggers? In pgAdmin there are directly listed under the database-menu.

![grafik](https://user-images.githubusercontent.com/25656713/141792471-2d39ecbe-d4a4-436f-a02e-933bc35f1b4f.png)

But I couldn't find any menu in DBeaver, which would show me these event-triggers.

**Comment by LonwoLonwo** [comment]
Nope, sorry. We do not have the "Event triggers" tab for PostgreSQL.
You can create a ticket as a feature request here: https://github.com/dbeaver/dbeaver/issues
